### PR TITLE
implemented getTajpuriya function

### PR DESCRIPTION
**Pull request by Thepuja**

### CODE REVIEW

**Review by Thepuja — COMMENTED** [review]
fixed the error in line 33 and 26 changed file path

File: nepalingo-web/src/hooks/getTajpuriya.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import useSWR from 'swr';
+import { DictionaryProps, DictionaryResponse } from './useDictionary';
+import { parse } from 'papaparse';
+import fs from 'fs';
+interface WordRecord{
+  word: string;
+  translation: string;
+}
+const fetcher = (url: string) => {
+  const csvfilepath = fs.readFile('Tajpuriya grammer - Sheet1.csv','utf8');
```
Comment (christikaes) [inline nepalingo-web/src/hooks/getTajpuriya.tsx:10]: This is the server side fetching logic that I mentioned in our 1-1 yesterday. Use the `fetch` function instead

File: nepalingo-web/src/hooks/getTajpuriya.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      {
+      
+        meaningOriginal: wordData.meaning_tajpuriya,
+        meaningEn: wordData.meaning_en,
+      },
+    ],
+  };
+};
+
+const getTajpuria = (props: Omit<DictionaryProps, 'language'>) => {
```
Comment (christikaes) [inline nepalingo-web/src/hooks/getTajpuriya.tsx:33]: Here we shouldn't omit the language - your response has language on like 37 so you don't need this

Comment (christikaes) [inline nepalingo-web/src/hooks/getTajpuriya.tsx:33]: This should be renamed to `useTajpuria` since it is a hook (there is a useSWR inside of it)

**Review by christikaes — CHANGES_REQUESTED** [review]
This looks pretty close! Nice job. I left a few comments

File: nepalingo-web/src/hooks/getTajpuriya.tsx
Comment (NancyAanchal) [inline nepalingo-web/src/hooks/getTajpuriya.tsx:1]: const sourceFile = "./Tajpuriya grammer - Sheet1.csv";

for this file path to work, u should move ur csv file to the public folder. 
make a 'dictionaries' folder in public and put ur csv file in it. then change the file path to  './dictionaries/Tajpuriya grammer - Sheet1.csv'

**Review by christikaes — APPROVED** [review]
Thanks for fixing these changes!

**Pull request merged**